styles/nav/_styles.scss

```diff
@@ -0,0 +1,88 @@
+//== Nav/Subnav
+//
+//## Styles for Tab Navigation and Sub-Navigation
+
+.daptiv-nav {
+    background: $nav-bg;
+    font: $nav-font;
+    height: $nav-height;
+    line-height: $nav-height;
+    width: 100%;
+
+    .tab {
+        color: $nav-font-color;
+        display: inline-block;
+        padding: 0 $nav-tab-h-pad;
+        text-decoration: none;
+
+        & > * {
+            vertical-align: middle;
+        }
+
+        &:not(.selected):hover {
+            color: $nav-font-color-hover;
+        }
+
+        &.selected {
+            background: $nav-tab-selected-bg;
+        }
+
+        &.menu::after {
+            @extend .fa;
+
+            content: $fa-var-caret-down;
+            font-size: $icon-size-small;
+            margin-left: $half-whitespace-unit;
+        }
+    }
+}
+
+/*
+    Sub Navigation tab
+*/
+.daptiv-subnav {
+    background: $subnav-bg;
+    font: $subnav-font;
+    height: $subnav-height;
+    line-height: $subnav-height;
+    width: 100%;
+
+    .tab {
+        color: $subnav-font-color;
+        padding: 0 $subnav-tab-h-pad;
+        text-decoration: none;
+
+        & > * {
+            vertical-align: middle;
+        }
+
+        &:not(.selected):hover {
+            color: $subnav-font-color-hover;
+        }
+
+        &.selected {
+            color: $subnav-tab-selected-color;
+        }
+
+        &.disabled {
+            color: $color-medium-dark-gray;
+            cursor: default;
+
+            &:hover {
+                color: $color-medium-dark-gray;
+            }
+        }
+
+        &.menu {
```

Comment:
also, this isn't the dropdown menu, this is just the styling for the item which indicates a menu is available (menu tab in this case specifically) which is why it was moved to iconography. Here it was actually redefining the `icon` class and also defining a `menu` type of icon class. All of which pointed to it being an icon style and not a nav dependent style.